nihms-native-assembler/src/test/java/org/dataconservancy/nihms/assembler/nihmsnative/NihmsMetadataSerializerTest.java

```diff
@@ -129,6 +135,7 @@ public void testSerializedMetadataValidity() throws Exception {
 
         OutputStream os = new FileOutputStream(targetFile);
```

Comment:
I know this isn't a part of this PR, but I'm wondering if this PR could be included to put `MetadataSerializerTest.xml` under `target/`, or better yet use something like `File.createTmpFile("NihmsMetadataSerializerTest-", ".xml")` for the output file.

If the assertions pass, then the output file can be removed.  If the assertions about the file fail, then the file can be kept, and the path to the file echoed out in the assertion error message.

This could be done in a separate PR if you wanted to, too.  Doesn't have to be in this one.  Putting the file under `target` or in a tmp dir will help keep git happy (`git status` won't report a file that needs to be added, for example).

Reply:
> equals on metadata is not suited for field value comparison at the moment it seems.

@jrmartino in what way?  We can't do a value-by-value comparison of the fields?